### DISCUSSION TITLE
Several AzuraCast Questions

### DISCUSSION THREAD

**Discussion by bombelman** [AzuraCast/AzuraCast#7206] [Q&A]
Hi all, I recently started with AzuraCast.
I have used several other stream platforms before, and also the plain-jane SC DNAS also.
Here I have some general questions about AzuraCast.
Because I'm not sure if it is a missing feature, I will post it here first, and can request it as a feature/update/bug if it is not how it should be. Feel free to link me to a relevant doc/man page.
At time of writing (June 16th) I'm running v0.20.1 Stable (Docker). The install is today 2 months old.
In all cases, I use Shoutcast / DNAS (not Icecast).

Question 1: The advanced tab when editing mount point has the option for a (custom) url. I can't seem to get this to work. What are acceptable values that are possible ? Working examples would be great !
e.g. I currently use: my.domain.com/listen/stationname/mountpoint
Can I get: something like my.domain.com/stationname ?

Question 2: Has anyone got the ban-IP feature to work ? Ip's in the list are not blocked and still connect.
(Issue already opened here btw: https://github.com/AzuraCast/AzuraCast/issues/7194)

Question 3: With AutoDJ feature: How/where do I make/manage a custom playlist with tracks in my order ?

Question 4: What exactly is the "Browser URL" feature on the System settings page ? 
Could not find it in docs.

Question 5: Can I use 1 license of StereoTool for my complete AzuraCast or do I need a separate paid license for every station ?

Thanks everyone, much appreciated.

**Comment by Heimatradio** [comment]
Hi,
Question 4: This is your domain that points to the Azuracast VPS, e.g. https://streams.yourdomain.com

Question 5: This can work with one licence, the StereoTool license check server only checks whether it is a valid license or a valid username (from the license). My personal decision would be to have a separate basic license for each station.

Greetings

**Comment by LordHelmchen666** [comment]
1. You can use it as a "redirect". It is the listen_url. My stream would be https://stream.eisradio.de/listen/eisradio/lautfm.mp3 but my listeners should use https://eisradio.stream.laut.fm/eisradio. So i put this in custom URL. see it here https://dir.xiph.org/search?q=eisradio click on Play (Eisradio)
Nowplaying API looks like this
```
{
  "station": {
    "id": 1,
    "name": "Eisradio",
    "shortcode": "eisradio",
    "description": "Eisradio - Sport Radio und Community aus Regensburg (Oberpfalz/ Bayern).",
    "frontend": "icecast",
    "backend": "liquidsoap",
    "timezone": "Europe/Berlin",
    "listen_url": "https://eisradio.stream.laut.fm/eisradio",
    "url": "https://www.eisradio.de/",
    "public_player_url": "https://stream.eisradio.de/public/eisradio",
    "playlist_pls_url": "https://stream.eisradio.de/public/eisradio/playlist.pls",
    "playlist_m3u_url": "https://stream.eisradio.de/public/eisradio/playlist.m3u",
    "is_public": true,
    "mounts": [
      {
        "id": 11,
        "name": "Laut.fm Eisradio",
        "url": "https://eisradio.stream.laut.fm/eisradio",
        "bitrate": 192,
        "format": "mp3",
        "listeners": {
          "total": 0,
          "unique": 0,
          "current": 0
        },
        "path": "/lautfm.mp3",
        "is_default": true
      }
    ],
    "remotes": [
      {
        "id": 12,
        "name": "LautFM Eisradio Stream",
        "url": "https://eisradio.stream.laut.fm/eisradio",
        "bitrate": 192,
        "format": "mp3",
,,,,
```

2. Had the same problem but i don't need it any longer.
3. Add Playlist
 - Song Playback Order check Sequential (The order of the playlist is manually specified and followed by the AutoDJ.)
 - add tracks to playlist
 - go back to playlist
 - click on More
 - click on Reorder
 - a modal opens where you can order tracks
